cmdstanpy/stanfit/gq.py

```diff
@@ -633,11 +626,10 @@ def _assemble_generated_quantities(self) -> None:
             order='F',
         )
         for chain in range(self.chains):
-            with open(self.runset.csv_files[chain], 'r') as fd:
-                lines = (line for line in fd if not line.startswith('#'))
-                gq_sample[:, chain, :] = np.loadtxt(
-                    lines, dtype=np.ndarray, ndmin=2, skiprows=1, delimiter=','
-                )
+            *_, draws = stancsv.parse_comments_header_and_draws(
```

Comment:
Same as mcmc, probably want a `try`/re-raise with specific chain name here.

Reply:
I think this is a good idea -- even in the single csv file case, it would be useful to at least identify the file location that cause the issue. If nothing else, it would probably speed up debugging in some unknown instances where errors are thrown.

I've added `try...except`s around the various places we process the stan csvs to raise the problem file path from the original exception.